Standardize output structure and configuration

eletonia opened the pull request:
# Summary

 This PR makes the following changes to runner:

- simplifies the output directory structure so results are now saved in `<user>/<hostname>/<btim>/<etim>` instead of the long directory structure with all the other parameters we were using earlier
- outputs the parameters in `params.json` in `<user>/<hostname>/<btim>` instead of the params text file for easy reading of the file

*note that this introduces jq as a dependency for runner

# GitHub Issue(s) to reference

 - #23 

# Reminders
 * [x] should this PR noted in the Changelog?

WarningRan commented:
It seems like the commits don't have signature, could you please add it? And could you update [changelog](https://github.com/IBM/fmwork/blob/main/CHANGELOG.md) with your new feature? Thank you @eletonia !

eletonia commented:
Hi @WarningRan
I updated change log with the update. I tried to sign off on my commits and push it but I can't force push to this branch since it's a protected branch. I think the admin of this repo needs to allow it - https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/managing-protected-branches/managing-a-branch-protection-rule
```
$ git rebase HEAD~5 --signoff
Current branch lastmile is up to date, rebase forced.
Successfully rebased and updated refs/heads/lastmile.
$ git push --force-with-lease origin lastmile
Enumerating objects: 28, done.
Counting objects: 100% (28/28), done.
Delta compression using up to 128 threads
Compressing objects: 100% (10/10), done.
Writing objects: 100% (23/23), 2.54 KiB | 2.54 MiB/s, done.
Total 23 (delta 13), reused 13 (delta 9), pack-reused 0
remote: Resolving deltas: 100% (13/13), completed with 3 local objects.
remote: error: GH006: Protected branch update failed for refs/heads/lastmile.
remote: 
remote: - Cannot force-push to this branch
To github.com:IBM/fmwork.git
 ! [remote rejected] lastmile -> lastmile (protected branch hook declined)
error: failed to push some refs to 'github.com:IBM/fmwork.git'
```

WarningRan commented:
Thank you for the changes on the log, @eletonia ! 
If reaching out to an admin (Nelson) to allow force-push isn’t ideal, here’s a safe workaround:

1. **Create a new branch from `lastmile`:**

   ```bash
   git checkout -b lastmile-signed
   ```

2. **Rebase the last few commits with `--signoff`**  
   *(Adjust the number if needed)*

   ```bash
   git rebase HEAD~5 --signoff
   ```

3. **Push the new branch:**

   ```bash
   git push origin lastmile-signed
   ```

4. **Then either:**
   - Update your existing PR to use `lastmile-signed`, **or**
   - Create a new PR from that branch.

eletonia commented:
@nelsonspbr allowed force push so I think we're good now, the commits are signed off @WarningRan
